packages/integration-tests/test-config.js

```diff
@@ -18,6 +18,7 @@ require('dotenv').config()
 const config = {
   geth_url: process.env.GETH_URL || 'http://localhost:8545',
   watcher_url: process.env.WATCHER_URL || 'http://localhost:7434',
+  watcher_proxy_url: process.env.WATCHER_PROXY_URL || 'http://localhost:9000',
```

Comment:
Not sure if a default value is a good idea. I imagine that most test runs won't use a proxy

Reply:
shouldn't have a default value tho.